cmd/buildctl/main.go

```diff
@@ -57,6 +59,11 @@ func main() {
 			Usage: "client key",
 			Value: "",
 		},
+		cli.IntFlag{
```

Comment:
durationflag?

Reply:
I wanted exactly that and failed to find it, thanks!

Reply:
OTOH, that requires the user to write `10s` instead of `10` and provides all sorts of unnecessary flexibility (e.g. I don't think `42ms` is a useful thing to be able to specify).

I think maybe a basic seconds timeout is probably better.

Reply:
500ms might be still useful? (0.5 in float?)

Reply:
Is it useful enough to warrant the overhead/complexity in the common case (whole numbers of seconds) though?

I suppose I could make it a float, but ultimately does it really matter for a tool like `buildctl`? (I only really did this at all to be able validate the changes I made to `client`).

Reply:
(I accidentally pushed a switch to `DurationFlag` which I hadn't intended to make yet while fixing the test failures, I've undone while we discuss)